quinn-proto/src/connection/mtud.rs

```diff
@@ -81,10 +77,17 @@ impl MtuDiscovery {
     }
 
     /// Notifies the [`MtuDiscovery`] that a packet has been ACKed
-    pub(crate) fn on_acked(&mut self, space: SpaceId, packet_number: u64, packet_bytes: u16) {
+    ///
+    /// Returns true if the packet was an MTU probe
```

Comment:
nit: should the change to return a value here be part of the next commit? I don't see anywhere black hole detection uses it. No big deal though.

Reply:
If it's not big deal, I'd rather concentrate on the rest of the feedback. I'll keep it in mind for next PRs, though!